src/deploy/functions/backend.ts

```diff
@@ -133,6 +133,11 @@ export interface TargetIds {
 
 export type FunctionsPlatform = "gcfv1" | "gcfv2";
 
+/**
+ * Invoker access control type for https functions.
+ */
+export type Invoker = "public" | "private" | string;
```

Comment:
Same typing comments from firebase-functions - I prefer making this include the array type

Reply:
resolving this since we agreed to leave as-is for easier future development.